Content.Client/Shuttles/BUI/ShuttleConsoleBoundUserInterface.cs

```diff
@@ -66,6 +67,11 @@ private void OnUndockPressed(NetEntity obj)
         SendMessage(new UndockRequestMessage() { DockEntity = obj });
     }
 
+    private void OnThrustLimited(float val)
+    {
+        SendMessage(new ThrustLimitedMessage() { ThrustLimit = val });
+    }
+
```

Comment:
Is this sent on mouse release or every frame

Reply:
every frame where the thrust slider's value has changed, it's highly inconvenient if only sent on mouse release